pkg/testutils/mock_config.go

```diff
@@ -14,11 +15,34 @@ const (
 	namespace     = "kubevirt"
 )
 
-func NewFakeClusterConfig(cfgMap *v1.ConfigMap) (*virtconfig.ClusterConfig, cache.SharedIndexInformer) {
+func NewFakeClusterConfig(cfgMap *v1.ConfigMap) (*virtconfig.ClusterConfig, cache.SharedIndexInformer, cache.SharedIndexInformer) {
 	informer, _ := NewFakeInformerFor(&v1.ConfigMap{})
+	crdInformer, _ := NewFakeInformerFor(&extv1beta1.CustomResourceDefinition{})
+
 	copy := copy(cfgMap)
 	informer.GetStore().Add(copy)
-	return virtconfig.NewClusterConfig(informer, namespace), informer
+	crdInformer.GetStore().Add(&extv1beta1.CustomResourceDefinition{
```

Comment:
you could use `AddDataVolumeAPI` here

Reply:
yup, I'll make this change as well, thanks!